agent/consul/server.go

```diff
@@ -861,6 +894,44 @@ func (s *Server) setupRPC() error {
 	return nil
 }
 
+// setupGRPC initializes the built in gRPC server components
+func (s *Server) setupGRPC() error {
+	lis := &grpcListener{
+		addr:  s.Listener.Addr(),
+		conns: make(chan net.Conn),
+	}
+
+	// We don't need to pass tls.Config to the server since it's multiplexed
+	// behind the RPC listener, which already has TLS configured.
+	srv := grpc.NewServer(
+		grpc.StatsHandler(grpcStatsHandler),
+		grpc.StreamInterceptor(GRPCCountingStreamInterceptor),
+	)
+	//stream.RegisterConsulServer(srv, &ConsulGRPCAdapter{Health{s}})
```

Comment:
should this commented code be removed?

Reply:
It is in a later commit once the health endpoint actually exists again!